scripts/dependency-graph.js

```diff
@@ -1,189 +1,153 @@
-const Arborist = require('@npmcli/arborist')
-const { readFileSync } = require('fs')
-const { join } = require('path')
-const log = require('proc-log')
-const { run, CWD, pkg, fs } = require('./util.js')
+const Arborist = require("@npmcli/arborist");
+const { readFileSync, writeFile } = require("fs");
+const { join } = require("path");
 
 // Generates our dependency graph documents in DEPENDENCIES.md.
 
 // To re-create npm-cli-repos.txt run:
 // npx -p @npmcli/stafftools gh repos --json | json -a name | sort > scripts/npm-cli-repos.txt
-const repos = readFileSync(join(CWD, 'scripts', 'npm-cli-repos.txt'), 'utf-8').trim().split('\n')
+const repos = readFileSync(join(__dirname, "npm-cli-repos.txt"), "utf-8").trim().split("\n");
```

Comment:
This won't work, util.CWD is more than `__dirname` and it exists for a reason, for scripts to find their CWD.